docs/docs/cmd/spo/tenant/tenant-applicationcustomizer-get.md

```diff
@@ -0,0 +1,142 @@
+# spo tenant applicationcustomizer get
+
+Get an application customizer that is installed tenant wide
+
+## Usage
+
+```sh
+spo tenant applicationcustomizer get [options]
+```
+
+## Options
+
+`-t, --title [title]`
+: The title of the Application Customizer. Specify either `title`, `id`, or `clientSideComponentId`.
+
+`-i, --id [id]`
+: The product id of the Application Customizer. Specify either `title`, `id`, or `clientSideComponentId`.
+
+`-c, --clientSideComponentId  [clientSideComponentId]`
+: The Client Side Component Id (GUID) of the application customizer. Specify either `title`, `id`, or `clientSideComponentId`.
+
+--8<-- "docs/cmd/_global.md"
+
+## Examples
+
+Retrieves an application customizer by title.
+
+```sh
+m365 spo tenant applicationcustomizer get --title 'Some customizer'
```

Comment:
Let's use double quotes for all values to avoid issues across the different shells and how they parse quotes